redis/paymentCache.ts

```diff
@@ -141,7 +141,7 @@ export const getCachedPaymentsForUser = async (userId: string): Promise<Payment[
 }
 
 export const getCachedPaymentsCountForUser = async (userId: string): Promise<number> => {
-  const dashboardData = await getUserDashboardData(userId)
+  const dashboardData = await getUserDashboardData(userId, moment.tz.guess())
```

Comment:
this doesn't seem right. It will be run in the server. I tested adding 
```
  console.log('WIP tz', moment.tz.guess(), typeof window, 'is window type')
```
right above this line and got
```
2|next  | 2025-01-07T00:45:35: WIP tz Africa/Abidjan undefined is window type
```

... as a result, which shows:
- guess is wrong
- is being run in the server, since this was in the **server** logs (not the browser console) and even shows that `typeof window === undefined`.